dhis-2/dhis-web/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/CrudControllerAdvice.java

```diff
@@ -84,6 +85,11 @@
 @ControllerAdvice
 public class CrudControllerAdvice
```

Comment:
This Advice is only applied  to `MetadataSyncController` and `MetadataVersionController` so the "sensitive exception" logic would not be applied to Tracker import - which I guess was the original trigger for this PR.

Reply:
I am confused now. ControllerAdvice is applied globally across all controllers unless explicitly limited correct? 
Why do you think this only applies to those 2 controllers? 
I have also tested them on `TrackedEntityInstanceController`  and the exception handlers are handling the exceptions for that as well.

Reply:
You are absolutely right: 
> By default, the methods in an @ControllerAdvice apply globally to all controllers

I got confused by the fact that we have a couple of controllers extending `CrudControllerAdvice`, which, could probably be corrected.